.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea you'd like to see in this project
+title: "[FEATURE] I would like ..."
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
```

Comment:
I'd rather see the user describe the outcome they'd like, not the solution. Implementation is up to developers and as long as the right thing pops out at the end that's good enough for the user.

Reply:
Yeah, I meant that, just bad wording😅